e2e/ansible/playbooks/hyperconverged/test-k8s-crunchy-postgres/k8s-crunchy-pg.yml

```diff
@@ -73,12 +82,12 @@
          delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
 
        - name: Confirm the postgres statefulset is running 
-         shell: source ~/.profile; kubectl get pods | grep pgset
+         shell: source ~/.profile; kubectl get pods | grep pgset| grep Running | wc -l
```

Comment:
Get pods using the **selector/label** to get specific group of pods.

Reply:
Completely agree with you. Added steps to fetch app label and filter by it when looking for running replicas